Header header
whycon/detection_results[] tracked_objects          # A list of detected objects from a single timestamp